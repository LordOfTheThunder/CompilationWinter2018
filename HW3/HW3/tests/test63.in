void checkIfElse() {
	if (true) {
		if (true) {
			int x;
		}
		else {
			int y;
		}
	}
	else {
		int x;
		int y;
	}
}



void main() { print("I hate this course"); }



