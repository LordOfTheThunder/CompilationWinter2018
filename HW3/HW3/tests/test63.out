---end scope---
x INT 0
---end scope---
y INT 0
---end scope---
---end scope---
---end scope---
x INT 0
y INT 1
---end scope---
---end scope---
---end scope---
print (STRING)->VOID 1
printi (INT)->VOID 2
checkIfElse ()->VOID 3
